javascript/javascript1/week1/startup-Name.js

```diff
@@ -0,0 +1,9 @@
+const firstWords= ["Easy", "Awesome", "Corporate"];
+const secondWords=["Pease","Lemone","Squeezy"];
+let startupName="";
```

Comment:
You can directly declare the variable and assign its value instead of assigning an empty string here and then the actual value down